src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculation = {
+    result: 0,
+
+    add(value) {
+      calculation.result += value;
+    },
+
+    subtract(value) {
+      calculation.result -= value;
+    },
+
+    multiply(value) {
+      calculation.result *= value;
+    },
+
+    divide(value) {
+      calculation.result /= value;
+    },
+
+    reset() {
+      calculation.result = 0;
```

Comment:
Replace 'calculation.result' with 'this.result' to access the correct scope

Reply:
👌 fixed